src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -735,9 +737,20 @@ bool Dataloader::_load_map_dir(GameManager& game_manager) const {
 		ret = false;
 	}
 
-	if (!map.load_region_file(parse_defines(lookup_file(append_string_views(map_directory, region))).get_file_node())) {
-		Logger::error("Failed to load region file!");
-		ret = false;
+	{
+		std::vector<colour_t> colours;
+		if (!Map::load_region_colours(parse_defines(lookup_file(append_string_views(interface_directory, region_colours))).get_file_node(), colours)) {
```

Comment:
```suggestion
		if (!Map::load_region_colours(parse_defines(lookup_file(region_colours)).get_file_node(), colours)) {
```
Using a single variable for the region colours filepath saves us an unnecessary `append_string_views`, which currently allocates memory for an intermediate `std::string` holding the combined path.